libcontainerd/client_windows.go

```diff
@@ -264,12 +264,12 @@ func (clnt *client) Create(containerID string, checkpoint string, checkpointDir
 
 // AddProcess is the handler for adding a process to an already running
 // container. It's called through docker exec.
```

Comment:
Should we update the description of this function to mention it returns the PID?